integration/src/test/java/org/mule/test/integration/exceptions/ValidationModuleErrorHandlingTestCase.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.integration.exceptions;
+
+import io.qameta.allure.Feature;
+import io.qameta.allure.Issue;
+import io.qameta.allure.Story;
+import org.junit.Test;
+import org.mule.runtime.api.message.Message;
+import org.mule.test.AbstractIntegrationTestCase;
+import org.mule.tests.api.TestQueueManager;
+
+import javax.inject.Inject;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.mule.test.allure.AllureConstants.ErrorHandlingFeature.ERROR_HANDLING;
+import static org.mule.test.allure.AllureConstants.ErrorHandlingFeature.ErrorHandlingStory.ERROR_HANDLER;
+import static org.mule.test.allure.AllureConstants.ErrorHandlingFeature.ErrorHandlingStory.ERROR_MAPPINGS;
+
+@Feature(ERROR_HANDLING)
+@Story(ERROR_HANDLER)
+@Story(ERROR_MAPPINGS)
+public class ValidationModuleErrorHandlingTestCase extends AbstractIntegrationTestCase {
+
+  @Inject
+  private TestQueueManager queueManager;
+
+  @Override
+  protected String getConfigFile() {
+    return "org/mule/test/integration/exceptions/validation-module-error-handling.xml";
+  }
+
+  @Test
+  @Issue("MULE-19139")
+  public void validationAllWithErrorMapping() throws Exception {
+    flowRunner("validationAllWithErrorMapping").run();
```

Comment:
why use a queue and an error handler in the flow instead of just using the `runExpecting...` methods?